docs/en/observability/apm/manage-storage/ilm-how-to.asciidoc

```diff
@@ -134,7 +134,7 @@ This is not considered a breaking change as index management should continually
 [id="apm-data-streams-custom-policy{append-legacy}"]
```

Comment:
Sorry, I couldn't comment on the actual lines but by default apm-data plugin now doesn't ship with ILM but rather with [DSL](https://www.elastic.co/guide/en/elasticsearch/reference/current/data-stream-lifecycle.html). While the guide to create ILM and override DSL configs stays the same, we will have to fix the wordings a bit here.